examples/Example14/AdeptIntegration.cpp

```diff
@@ -0,0 +1,280 @@
+// SPDX-FileCopyrightText: 2021 CERN
```

Comment:
We've arrived in 2022. Some other files in this PR still have 2020, no sure where this was copied from (`example13` is correctly marked with 2021)...